adopt-a-dog/dog.html

```diff
@@ -0,0 +1,109 @@
+             <div class="adopt">Adopt</div>
+           </div>
+
+          <div class="thumbnail">
+            <img src="images/nikolay-tchaouchev-785309-unsplash.jpg" alt="Poppy" width="10%">
+            <h4>Poppy</h4>
+            <p>Cost to Adopt: $123.45</p>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+                tempor incididunt ut labore et dolore magna aliqua.</p>
+              <div class="adopt">Adopt</div>
+          </div>
+
+          <div class="thumbnail">
+            <img src="images/marcus-wallis-471438-unsplash.jpg" alt="Jack" width="10%">
+            <h4>Jack</h4>
+            <p>Cost to Adopt: $123.45</p>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+               tempor incididunt ut labore et dolore magna aliqua.</p>
+            <div class="adopt">Adopt</div>
+          </div>
+
+          <div class="thumbnail">
+            <img src="images/duffy-brook-350225-unsplash.jpg" alt="Duffy" width="10%">
+            <h4>Duffy</h4>
+            <p>Cost to Adopt: $123.45</p>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+               tempor incididunt ut labore et dolore magna aliqua.</p>
+            <div class="adopt">Adopt</div>
+          </div>
+
+          <div class="thumbnail">
+            <img src="images/tucker-good-589776-unsplash.jpg" alt="Lucas" width="10%">
+            <h4>Lucas</h4>
+            <p>Cost to Adopt: $123.45</p>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+             tempor incididunt ut labore et dolore magna aliqua.</p>
+             <div class="adopt">Adopt</div>
+           </div>
+
+           <div class="thumbnail">
+             <img src="images/mitchell-orr-237216-unsplash.jpg" alt="Jake" width="10%">
+             <h4>Jake</h4>
+             <p>Cost to Adopt: $123.45</p>
+             <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+              tempor incididunt ut labore et dolore magna aliqua.</p>
+              <div class="adopt">Adopt</div>
+          </div>
+
+          <div class="thumbnail">
+              <img src="#" alt="Angus" width="10%">
+              <h4>Angus</h4>
+              <p>Cost to Adopt: $123.45</p>
+              <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+               tempor incididunt ut labore et dolore magna aliqua.</p>
+               <div class="adopt">Adopt</div>
+            </div>
+
+            <div class="thumbnail">
+              <img src="#" alt="Violet" width="10%">
+              <h4>Violet</h4>
+              <p>Cost to Adopt: $123.45</p>
+              <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+               tempor incididunt ut labore et dolore magna aliqua.</p>
+               <div class="adopt">Adopt</div>
+             </div>
+
+          <footer>
```

Comment:
you're going to run into problems with the footer in the div of dogs. Make sure it's out of that parent wrapper